README.md

```diff
@@ -1 +1,17 @@
 # @bigtest/server
+
+To run the server:
+
+``` sh
+yarn watch yarn start
+```
+
+Then open the following URL in a browser:
+
+<http://localhost:24004/index.html?orchestrator=ws%3A%2F%2Flocalhost%3A24003>
+
+To run the tests:
+
+```
+yarn test
+```
```

Comment:
Nice! Even a little bit of documentation goes a long way.

Makes me wonder though, if we should save the typing of `yarn watch yarn start` each time and just roll the watching into a separate command. Even though the `watch` command can technically watch any process, are we really going to use it for watching anything else than `yarn start`?